Program.cs

```diff
@@ -10,40 +10,75 @@ internal class Program
     {
         static void Main(string[] args)
         {
-            string op;
-
-            Multiply multiply = new Multiply();
-            Add add = new Add();
-            Subtraction subtraction = new Subtraction();
-            Division division = new Division();
-            PierwszaLiczba pierwszaLiczba = new PierwszaLiczba();
-            Console.WriteLine("Wybierz działanie: \n" + "(+) :dodawanie \n" + "(-) :odejmowanie \n" + "(*) : mnożenie \n" + "(/) : dzielenie \n" + "l - liczby pierwsze" );
-            op = Console.ReadLine();
-            Console.WriteLine("Wpisz dwie dowolne liczby: ");
-           
-
-            if (op == "+")
             {
-                Console.WriteLine(add.add());
-            }
-            else if (op == "-")
-            {
-                Console.WriteLine(subtraction.subtr());
-            }
-            else if (op == "*")
-            {
-                Console.WriteLine(multiply.multiply());
-            }
-            else if (op == "/")
-                    {
-                Console.WriteLine(division.div());
-            }
-            else if (op == "l")
-            {
-                Console.WriteLine(pierwszaLiczba.pierwsza());
-            } 
+                string op;
 
-            Console.ReadLine();
+
+                Calculator calculator = new Calculator();
+
+                Console.WriteLine("Wybierz działanie: \n"
+                    + "(1) :dodawanie \n"
+                    + "(2) :odejmowanie \n"
+                    + "(3) : mnożenie \n"
+                    + "(4) : dzielenie \n"
+                    + "(5) - liczby pierwsze\n"
+                    + "(6) - silnia\n"
+                    + "(7) - liczba doskonała\n"
+                    + "(8) - potęga\n"
+                    + "(9) - średnia liczba z tablicy\n"
+                    + "(0) - max z tablicy \n"
+                    + "(*) - min z tablicy \n");
+                op = Console.ReadLine();
```

Comment:
Słaba nazwa zmiennej op nic mi nie mówi, to samo zmienne typu a, b itd. Cały kod przerefactoruj pod kątem nazewnictwa